modules/@apostrophecms/schema/index.js

```diff
@@ -2343,16 +2343,18 @@ module.exports = {
   extendMethods(self, options) {
     return {
       getBrowserData(_super, req) {
-        if (!req.user) {
+        if (req.scene !== 'apos') {
           return;
         }
-        const browserOptions = _super(req);
```

Comment:
You can keep this code if you fix `module/index.js` to also look at `req.scene`. You should do that to avoid missing out on standard things.